src/components/inputs/TokenSearchInputSelectTokens.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+type Props = {
+  addresses: string[];
+  label: string;
+};
+
+const props = withDefaults(defineProps<Props>(), {
+  addresses: () => []
+});
+
+const emit = defineEmits<{
+  (e: 'click', address: string): void;
+}>();
+</script>
+<template>
+  <div
+    :class="[
+      'border border-gray-100 dark:border-gray-900 rounded-lg',
+      'bg-white dark:bg-gray-850',
+      'flex flex-wrap items-center gap-3',
+      'text-gray-900 dark:text-gray-400',
+      'overflow-x-auto',
+      'px-3'
+    ]"
```

Comment:
What is the reason for writing classes this way? It seems that you can use the default string, it makes no sense to use a dynamic parameter

```suggestion
  <div
    class="border border-gray-100 dark:border-gray-900 rounded-lg bg-white dark:bg-gray-850 flex flex-wrap items-center gap-3 text-gray-900 dark:text-gray-400 overflow-x-auto px-3"
```

Reply:
Yeah, I personally prefer some kind of grouping for readability, because I'm not really used tailwind a lot, but forcing order with [eslint-plugin-tailwindcss](https://www.npmjs.com/package/eslint-plugin-tailwindcss) would solve this I guess.